threatstream.json

```diff
@@ -7205,10 +7243,15 @@
                         "vps"
                     ]
                 },
+                "source": {
+                    "description": "Source of observable to import(It will be reflected on UI only when observable is imported without approval)",
```

Comment:
```suggestion
                    "description": "Source of observable to import (It will only be reflected on UI when observable is imported without approval)",
```

Reply:
Done